src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventActions-Content-Published.SummaryAdmin.cshtml

```diff
@@ -13,15 +13,17 @@
     if (availableVersionsCount != versionNumber) 
     {
         @T["|"]
-        <a asp-action="Restore" 
-           asp-controller="Content" 
-           asp-route-auditTrailEventId="@auditTrailEvent.Id"
-           data-title="@T["Restore to Version"]" 
-           data-message="@T["Are you sure you want to restore to version {0}?", versionNumber]" 
-           data-ok-text="@T["Yes"]" 
-           data-cancel-text="@T["No"]"
-           itemprop="UnsafeUrl RemoveUrl">
-            @T["Restore"]
-        </a>
+        <form method="post">
```

Comment:
This should be added automatically by `UnsafeUrl RemoveUrl` as long as the admin JS is included. Also in AuditTrailEventActions-Content-Removed.SummaryAdmin.cshtml.